src/scripts/main.js

```diff
@@ -1,3 +1,8 @@
 'use strict';
 
-// write code here
+const spiderHTML = document.querySelector('.spider');
+
+spiderHTML.style.position = 'relative';
```

Comment:
To center the spider image using the DOM element's properties, you need to calculate the position based on the dimensions of the 'wall' and 'spider' elements instead of using percentages.